src/code/z_camera.c

```diff
@@ -8,12 +8,24 @@ s32 Camera_ChangeModeFlags(Camera* camera, s16 mode, u8 flags);
 s32 Camera_QRegInit(void);
 s32 Camera_UpdateWater(Camera* camera);
 
-#define RELOAD_PARAMS \
-    (camera->animState == 0 || camera->animState == 0xA || camera->animState == 0x14 || R_RELOAD_CAM_PARAMS)
+#define CAM_GET_RO_DATA(type, paramData) &((type*)paramData)->roData // Read-Only Data
+#define CAM_GET_RW_DATA(type, paramData) &((type*)paramData)->rwData // Read-Write Data
 
-#define PCT(x) ((x)*0.01f)
-#define NEXTSETTING ((values++)->val)
-#define NEXTPCT PCT(NEXTSETTING)
+// Camera will reload its paramData. This is mainly read-only data from stored in CameraModeValue. Although some
+// read-write data is reset as well
```

Comment:
I feel bad suggesting phrasing things but it's bothering me 😅 
```suggestion
// Camera will reload its paramData. Usually that means setting the read-only data from what is stored in CameraModeValue arrays. Although sometimes some
// read-write data is reset as well
```

Reply:
Please, feel free to find ways to better phrase things for clarity!